ddtrace/internal/ci_visibility/api/ci_base.py

```diff
@@ -133,48 +146,58 @@ def __init__(
         self._source_file_info: Optional[CISourceFileInfo] = None
         self._coverage_data: Optional[CICoverageData] = None
 
-        # Currently unsupported:
-        self._is_benchmark = False
-
-    def __repr__(self):
+    def __repr__(self) -> str:
         return f"{self.__class__.__name__}({self.item_id})"
 
-    def _add_all_tags_to_span(self):
+    @_require_span
+    def _add_all_tags_to_span(self) -> None:
+        if self._span is None:
```

Comment:
@romainkomorndatadog Because of the decorator, we know for sure `self._span` is not None here, no?